internals/secrethub/service_aws_init.go

```diff
@@ -81,8 +146,8 @@ func (cmd *ServiceAWSInitCommand) Run() error {
 func (cmd *ServiceAWSInitCommand) Register(r Registerer) {
 	clause := r.Command("init", "Create a new AWS service account attached to a repository.")
 	clause.Arg("repo", "The service account is attached to the repository in this path.").Required().SetValue(&cmd.repo)
-	clause.Flag("kms-key-id", "ID of the KMS-key to be used for encrypting the service's account key.").Required().StringVar(&cmd.kmsKeyID)
-	clause.Flag("role", "ARN of the IAM role that should have access to this service account.").Required().StringVar(&cmd.role)
+	clause.Flag("kms-key-id", "ID of the KMS-key to be used for encrypting the service's account key.").StringVar(&cmd.kmsKeyID)
```

Comment:
It looks like we start sentences for other args/flags with `The XXX ...`. Consider adding that here and the line below too to be consistent.